src/popup/imageDetailModule.js

```diff
@@ -81,6 +81,43 @@ export function getHtmlAttribution(image) {
   return `<p style="font-size: 0.9rem;font-style: italic;">${imgLink}${creator}${licenseLink}${licenseImgLink}</p>`;
 }
 
+/**
+ * @desc Helper function to formatHTML
+ * @param {number} level - left space count.
+ */
+function formatHTMLMain(node, newLevel) {
+  let level = newLevel;
+  const indentBefore = new Array(level + 1).join('  ');
+  level += 1;
+  const indentAfter = new Array(level - 1).join('  ');
+  let textNode;
+
+  for (let i = 0; i < node.children.length; i += 1) {
+    textNode = document.createTextNode(`\n${indentBefore}`);
+    node.insertBefore(textNode, node.children[i]);
+
+    formatHTMLMain(node.children[i], level);
+
+    if (node.lastElementChild === node.children[i]) {
+      textNode = document.createTextNode(`\n${indentAfter}`);
+      node.appendChild(textNode);
+    }
+  }
+
+  return node;
+}
+
+/**
+ * @desc Helps in Formatting HTML Attributions
+ * @param {string} str - The image object.
+ */
+function formatHTML(str) {
+  const div = document.createElement('div');
+  div.innerHTML = str.trim();
+
+  return formatHTMLMain(div, 0).innerHTML;
+}
```

Comment:
Can we avoid using `innerHTML`? Because the firefox addon stores rejects the extension build if we use `innerHTML`.

> Warning: If your project is one that will undergo any form of security review, using innerHTML most likely will result in your code being rejected. For example, if you use innerHTML in a browser extension and submit the extension to addons.mozilla.org, it will not pass the automated review process.

https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML

Reply:
Done